netutils/config/compliance.py

```diff
@@ -418,7 +430,7 @@ def section_config(feature, device_cfg, network_os):
                 continue
             else:
                 match = False
-        for line_start in section_starts_with:
+        for line_start in section_starts_with:  # type: ignore
```

Comment:
Could be fixed with the `feature` `TypedDict` as `feature.get("section")` would return something whose type Mypy would be able to infer correctly.